src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -133,11 +133,14 @@ public static void main(String[] args) throws Exception {
       // BEGIN LOADING WORKLOAD CONFIGURATION
       // ----------------------------------------------------------------
 
-      String geopartitionedConfigFile = options.getGeoPartitionedConfigFile().orElse("config/geopartitioned_workload.xml");
-      GeoPartitionedConfigFileOptions geopartitionedConfigOptions = new GeoPartitionedConfigFileOptions(geopartitionedConfigFile);
-      GeoPartitionPolicy geoPartitionPolicy = geopartitionedConfigOptions.getGeoPartitionPlacement(totalWarehousesAcrossShards, numWarehouses, startWarehouseIdForShard);
-      
+      String geopartitionedConfigFile = options.getGeoPartitionedConfigFile().orElse("");
```

Comment:
I think we should keep this code as is for the cases when the default file is available. We can rather do a file exists check for the geopartitionedConfigFile variable.